hedera-node/hedera-app/src/test/java/com/hedera/node/app/state/merkle/memory/InMemoryWritableStateTest.java

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.node.app.state.merkle.memory;
+
+import com.hedera.node.app.state.merkle.MerkleTestBase;
+
+class InMemoryWritableStateTest extends MerkleTestBase {
+
+    //    @Nested
+    //    @DisplayName("Constructor Tests")
+    //    final class ConstructorTest {
+    //        @Test
+    //        @DisplayName("You must specify the metadata")
+    //        void nullMetadataThrows() {
+    //            assertThatThrownBy(() -> new InMemoryWritableState<>(null, fruitMerkleMap))
+    //                    .isInstanceOf(NullPointerException.class);
+    //        }
+    //
+    //        @Test
+    //        @DisplayName("You must specify the merkle map")
```

Comment:
Can this be deleted?

Reply:
I have uncommented and fixed it up, so it is now part of our test suite!